Models/testmodel1/config.cpp

```diff
@@ -0,0 +1,49 @@
+#include "config.hpp"
+
+//TODO: Consider where these should be initialized
+TempPWMDriver PWMDriver;
+TempDSHOTDriver DSHOTdriver;
+
+Motor_t motors[] = {
+    {   //Yaw servo motor
+        .axis = yaw,
+        .direction = positive,
+        .driver = PWMDriver,
+        .interfaceID = 0
+    },
+    {   //Roll BLDC motor
+        .axis = roll,
+        .direction = negative,
+        .driver = DSHOTdriver,
+        .interfaceID = 0
+    }
+};
+
+const uint8_t NUM_MOTORS = sizeof(motors)/sizeof(Motor_t);
+
+PID_t flightmode1Tuning = {
+    .p = 1.0f,
+    .i = 1.0f,
+    .d = 1.0f
+};
```

Comment:
This unfortunately is unlikely to be sufficient. We will probably need PIDs for each axis that we are attempting to control. I'm not sure how best we handle this.

Maybe each flight mode has an enum of the axes it is controlling, then we define an array with PIDs for each axis in that enum?

Reply:
I expanded it now to apply to each axis in the same way the control limits do.

Reply:
I think it's possible that PIDs will be needed for axis that aren't just the physical orientation (e.g. speed, altitude?), so I'm not quite sure if that works either. It might be good enough for now though.